src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -621,8 +671,62 @@ protected String getRegionName(JcloudsLocation loc) {
     protected boolean isAvailabilityZone(String name) {
         return Strings.isNonBlank(name) && Character.isLetter(name.charAt(name.length()-1));
     }
-    
-    protected AmazonElasticLoadBalancingClient newClient(JcloudsLocation loc) {
+
+    private void applyLocationCustomizers(JcloudsLocation loc) {
+
+        final Collection<JcloudsLocationCustomizer> customizers = getCustomizers();
+        if (customizers != null) {
+            Template template = loc.buildTemplate(loc.getComputeService(), config().getBag(), customizers);
+            for (JcloudsLocationCustomizer customizer : customizers) {
+                customizer.customize(loc, loc.getComputeService(), template);
+            }
+        }
+    }
+
+    private void preReleaseLocationCustomizations() {
+        final Collection<JcloudsLocationCustomizer> customizers = getCustomizers();
+        if (customizers != null) {
+            for (final JcloudsLocationCustomizer customizer : customizers) {
+                safelyRelease(new Runnable() {
```

Comment:
I'd recommend making this runnable and the one in `postRelease` named inner classes to ensure there's no problem with persisted state and changes to this entity in the future (e.g. persisted state referring to `ElbControllerImpl$3` and that class changing due to ordering in source code).